src/interfaces/token.ts

```diff
@@ -64,21 +64,20 @@ export class TokenInterface<C extends ChainId> extends ServiceInterface<C> {
       .balances(address)
       .then(balances => balances.filter(token => token.balance !== "0"));
 
-    if (this.chainId === 1 || this.chainId === 1337) {
-      let zapperBalances = await this.yearn.services.zapper.balances(address);
-      const vaultBalanceAddresses = new Set(vaultBalances.map(balance => balance.address));
-      zapperBalances = zapperBalances.filter(balance => !vaultBalanceAddresses.has(balance.address));
-      return zapperBalances.concat(vaultBalances);
-    }
-
-    if (this.chainId === 250 || this.chainId === 42161) {
-      let ironBankTokens = await this.yearn.ironBank.balances(address);
-      const vaultBalanceAddresses = new Set(vaultBalances.map(balance => balance.address));
-      ironBankTokens = ironBankTokens.filter(balance => !vaultBalanceAddresses.has(balance.address));
-      return ironBankTokens.concat(vaultBalances);
-    }
-
-    throw new SdkError(`the chain ${this.chainId} hasn't been implemented yet`);
+    const vaultBalanceAddresses = new Set(vaultBalances.map(balance => balance.address));
+      const filterBalance = tokens => tokens.filter(balance => !vaultBalanceAddresses.has(balance.address));
+     let zapperBalances = await this.yearn.services.zapper.balances(address).filter(balance => !vaultBalanceAddresses.has(balance.address));
```

Comment:
we should not fetch zapper balances if we aren't on 1 or 1337 chain id

Reply:
ok got it, and soon I will fix it